src/tests/JIT/Directed/StructPromote/FldSeqsInPromotedCpBlk.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+class FldSeqsInPromotedCpBlk
```

Comment:
Can you add a comment linking this test to the PR and/or describing the problem this test is supposed to cover?